symengine/expression.h

```diff
@@ -120,12 +123,23 @@ class Expression
     {
         return eq(*m_basic, *other.m_basic);
     }
+
+    bool operator>=(const Expression &other) const
```

Comment:
Here's what the error looks like:
In file included from /Users/irislui/Documents/ECS_193/SymEngine/symengine/symengine/basic.cpp:12:
/Users/irislui/Documents/ECS_193/SymEngine/symengine/symengine/printer.h:65:51: error: invalid operands to
      binary expression ('const SymEngine::Expression' and 'int')
                if (it->first == 0 and it->second >= 0) {
                                           ~~~~~~~~~~ ^  ~